client/src/app/tabs/cloud-dmn/DmnEditor.js

```diff
@@ -0,0 +1,1023 @@
+/**
```

Comment:
The two dmn editors are 99% identical: https://www.diffchecker.com/mzSnaLJT

![image](https://user-images.githubusercontent.com/9433996/160371908-48dc8fa9-0445-4859-a4e0-dc41f0bf1aab.png)

But I guess there is still nothing we can do against (right now)

Reply:
Indeed, it's even worse than with BPMN. Some changes will come when we implement tracking and plugins, but still this is a situation which is far from ideal.